### PR TITLE
Add note to help text to mention necessary permissions

### PR DESCRIPTION
**Pull request by Benjamintf1**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing performed?

- [ ] Unit tests
- [ ] Integration tests
- [ ] Acceptance tests

## Checklist:

- [ ] This PR is being made against the `main` branch, or relevant version branch
- [ ] I have made corresponding changes to the documentation
- [ ] I have added testing for my changes

If you have any questions, or want to get attention for a PR or issue please reach out on the [#logging-and-metrics channel in the cloudfoundry slack](https://cloudfoundry.slack.com/archives/CUW93AF3M)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ OPTIONS:
    --guid              Display raw source GUIDs with no source Names. Incompativle with 'source' and 'source-type' for --sort-by. Incompatible with 'application' for --source-type
    --noise             Fetch and display the rate of envelopes per minute for the last minute. WARNING: This is slow...
    --sort-by           Sort by specified column. Available: 'source-id', 'source', 'source-type', 'count', 'expired', 'cache-duration', and 'rate'.
-   --source-type       Source type of information to show. Available: 'all', 'application', and 'platform'.
+   --source-type       "Source type of information to show. Available: 'all', 'application', 'service', 'platform', and 'unknown'. Excludes unknown sources unless 'all' or 'unknown' is selected, or `--guid` is used. To receive information on platform or unknown source id's, you must have the doppler.firehose, or logs.admin scope."
```
Comment (ctlong) [inline README.md:91]: I don't think that the `"` chars should be included at the beginning and end of this message.

**Review by ctlong — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Benjamintf1) [reply README.md]: fixed

**Review by ctlong — APPROVED** [review]
LGTM

**Comment by Benjamintf1** [comment]
oh no, branch permissions D:

**Comment by ctlong** [comment]
Raised the branch permissions issue with Amelia: https://cloudfoundry.slack.com/archives/C02HNDJB31R/p1666042497564869.

**Pull request merged**